assets/blocks/course-outline/lesson-block/lesson-block.scss

```diff
@@ -45,6 +45,18 @@
 		}
 	}
 
+	&__badge {
+		.wp-block-sensei-lms-course-outline-lesson & {
+			flex: 0 0 auto;
+			font-size: 0.6em;
+			color: $dark-gray;
```

Comment:
Let's keep the lesson text colors here with `color: inherit` and `border-color: currentColor` — just like the status. That makes it simpler to make this work with all color combinations, like a dark background.

Reply:
Good catch! Updated here: https://github.com/Automattic/sensei/pull/3732/commits/67a7991321694ff39476fa52a8a1587381e1496c